release-notes/opensearch.release-notes-2.1.0.md

```diff
@@ -0,0 +1,84 @@
+## 2022-07-05 Version 2.1.0 Release Notes
+
+
+### Breaking Changes in 2.1.0
+
+#### Remove Mapping types
+
+* [Type removal] Ignore _type field in bulk request ([#3505](https://github.com/opensearch-project/opensearch/pull/3505))
+* [Type removal] Remove type from BulkRequestParser (#3423) ([#3431](https://github.com/opensearch-project/opensearch/pull/3431))
+* [Type removal] _type removal from tests of yaml tests (#3406) ([#3414](https://github.com/opensearch-project/opensearch/pull/3414))
+
+#### Upgrades
+
+* [Upgrade] Lucene-9.3.0-snapshot-b7231bb (#3537) ([#3555](https://github.com/opensearch-project/opensearch/pull/3555))
+* [Upgrade] Lucene-9.2.0-snapshot-ba8c3a8 (#3416) ([#3422](https://github.com/opensearch-project/opensearch/pull/3422))
+* [Upgrade] Lucene-9.2-snapshot (#2924) ([#3419](https://github.com/opensearch-project/opensearch/pull/3419))
+
+
+#### Deprecations
+
+* [2.x] Deprecate public methods and variables with master term in package 'org.opensearch.action.support.master' (#3617) ([#3643](https://github.com/opensearch-project/opensearch/pull/3643))
+* Deprecate classes in org.opensearch.action.support.master (#3593) ([#3609](https://github.com/opensearch-project/opensearch/pull/3609))
+
+### Security Fixes
+
+* [Dependency upgrade] google-oauth-client to 1.33.3 (#3500) ([#3501](https://github.com/opensearch-project/opensearch/pull/3501))
+
+### Features/Enhancements
+
+* Restore renaming method onMaster() and offMaster() in interface LocalNodeMasterListener ([#3687](https://github.com/opensearch-project/opensearch/pull/3687))
+* Revert renaming masterOperation() to clusterManagerOperation() ([#3682](https://github.com/opensearch-project/opensearch/pull/3682))
+* Restore AcknowledgedResponse and AcknowledgedRequest to package org.opensearch.action.support.master (#3669) ([#3670](https://github.com/opensearch-project/opensearch/pull/3670))
+* Remove class org.opensearch.action.support.master.AcknowledgedResponse (#3662) ([#3668](https://github.com/opensearch-project/opensearch/pull/3668))
```

Comment:
Could you remove the L30-33? (PR 3668 3670 3682 2687)
They are used for reverting a few wrong changes, and those changes have not been released yet. 😁

Reply:
Sure thanks  @tlfeng , that makes sense to remove them as they are just revert of wrong changes